src/test/java/com/google/devtools/build/lib/actions/ActionCacheCheckerTest.java

```diff
@@ -655,15 +687,16 @@ public void saveOutputMetadata_treeMetadata_remoteFileMetadataSaved() throws Exc
             "file2", createRemoteFileMetadata("content2"));
     Action action =
         new InjectOutputTreeMetadataAction(
-            output, createTreeMetadata(output, children, Optional.empty()));
+            output, createTreeMetadata(output, children, Optional.empty(), Optional.empty()));
```

Comment:
nit: some `/*foo=*/` comments would aid readability here for the `Optional.empty()`.

Reply:
Done.